nncf/openvino/graph/model_utils.py

```diff
@@ -62,6 +62,29 @@ def get_start_nodes_for_activation_path_tracing(nncf_graph: NNCFGraph) -> List[N
     return nncf_graph.get_input_nodes() + nncf_graph.get_nodes_by_metatypes([OVReadValueMetatype])
 
 
+def remove_friendly_name_duplicates(model: ov.Model) -> ov.Model:
+    """
+    Removes diplicates of node names (friendly_name attribute) in the model.
+
+    :param model: ov.Model instance to update.
+    :return: Updated ov.Model without duplicated names.
+    """
+    rt_info_path = ["nncf", "friendly_names_were_updated"]
```

Comment:
it looks like idea with using `rt_info` will not work without copying `rt_info` in the model transformer. Could you check?

In general it would be a good idea not to lose the rt_info of the original model in the quantized model. I think this statement can be discussed offline in the team.

Reply:
Generally, our `ov.Model` creations from the `ModelTransformer` are isolated from the main model. But I've added `rt_info` copying as well for consistency.
`rt_info` can't be lost during the pipeline since we use the `ov.Model.clone()` method and we hope it correctly works with `rt_info`.